JSDemos/Demos/Tabs/Overview/Angular/app/app.component.css

```diff
@@ -1,26 +1,61 @@
-::ng-deep .content {
-  text-align: justify;
-  margin-top: 25px;
+::ng-deep #tabs-demo {
+  display: flex;
+  min-height: 450px;
+  justify-content: space-between;
 }
 
-::ng-deep #longtabs {
-  margin-top: 20px;
+::ng-deep .widget-container {
+  align-items: center;
+  flex-grow: 1;
+  padding: 16px 4px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
 }
 
-::ng-deep #scrolledtabs {
-  margin-top: 20px;
+::ng-deep .options {
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  padding: 20px;
+  width: 260px;
+  background-color: rgba(191, 191, 191, 0.15);
+}
+
+::ng-deep .tabs-container:not(:last-child) .dx-tabs-horizontal {
```

Comment:
Let's use [[gap (row-gap/column-gap)](https://developer.mozilla.org/en-US/docs/Web/CSS/gap)] instead of margin-bottom/right. In this case you shouldn't use :not(:last-child) selector.